reference/recency-surveillance.v1.json

```diff
@@ -59,150 +59,155 @@
                       "encounter_uuid"
                     ],
                     "properties": {
+                      "description": "DHIS2 organisation unit uuid for this facility",
                       "dhis2_orgunit_uuid": {
                         "type": "string",
                         "default": "text"
                       },
-                      "referred_location": {
+                      "encounter_uuid": {
+                        "description": "Encounter universal unique identifier",
                         "type": "string",
                         "default": "text"
                       },
-                      "referred_to_hiv_care": {
+                      "serial_number": {
+                        "description": "HTS number. This should be same number used in HTS blood samples sent to UVRI",
                         "type": "string",
-                        "enum": [
-                          "No",
-                          "Yes"
-                        ]
+                        "default": "text"
                       },
-                      "referred_to_tb_services": {
+                      "visit_date": {
+                        "description": "Must be YYYY-MM-DD format",
```

Comment:
@ODORA0 update the description to match the default format and also an example date to illustrate

Reply:
@ssmusoke Done